tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.scss

```diff
@@ -92,3 +104,53 @@ text {
     );
   }
 }
+
+.extent-edit-button {
+  background-color: mat-color($mat-gray, 200);
+  display: none;
+  font-size: 0;
```

Comment:
This element has no text, and line-height is fixed.  Checking: can `font-size` be dropped?

Reply:
I cannot fathom internals of mat-icon-button but without font-size, it pushes the mat-icon down to near ~line-height which makes me think that there is some invisible whitespace in the template causing the UI jank.